src/uikit.cs

```diff
@@ -4023,6 +4305,29 @@ interface UICollectionViewDelegate : UIScrollViewDelegate {
 		[Export ("collectionView:sceneActivationConfigurationForItemAtIndexPath:point:")]
 		[return: NullAllowed]
 		UIWindowSceneActivationConfiguration GetSceneActivationConfigurationForItem (UICollectionView collectionView, NSIndexPath indexPath, CGPoint point);
+
+		[Watch (9,0), TV (16,0), iOS (16,0), MacCatalyst (16,0)]
+		[Export ("collectionView:canPerformPrimaryActionForItemAtIndexPath:")]
+		bool CanPerformPrimaryActionForItemAtIndexPath (UICollectionView collectionView, NSIndexPath indexPath);
```

Comment:
```suggestion
		bool CanPerformPrimaryActionForItem (UICollectionView collectionView, NSIndexPath indexPath);
```
maybe can be shortened to this or even without the "ForItem"